src/bci_build/templates.py

```diff
@@ -153,6 +153,12 @@
     <param name="regex">{{ replacement.regex_in_build_description }}</param>
     <param name="package">{{ replacement.package_name }}</param>{% if replacement.parse_version %}
     <param name="parse-version">{{ replacement.parse_version }}</param>{% endif %}
+  </service>
+  <service name="replace_using_package_version" mode="buildtime">
+    <param name="file">README.md</param>
+    <param name="regex">{{ replacement.regex_in_build_description }}</param>
+    <param name="package">{{ replacement.package_name }}</param>{% if replacement.parse_version %}
+    <param name="parse-version">{{ replacement.parse_version }}</param>{% endif %}
```

Comment:
Please no, this is creating needless `_service` entries. Imho we should be doing something like https://github.com/SUSE/BCI-dockerfile-generator/pull/998/commits/905f710a3b9f0932088832e3c6a00f2e4607fefa instead

```suggestion
```

Reply:
Ok, agree, this was not the best solution. The best approach would be to change `Replacement` indeed.

I can add a `file` attribute to specify the file we want to replace something, but this affects the older images which is not ideal. Unless we Docker/Kiwi template by default, but that can have unwanted side effects one day.

I can add a `regex_in_readme` attribute, this will only affect who uses this, and automatically selects README.md, similar to `regex_in_build_description`.

Which one sounds better, or do you have a better idea?

Reply:
I picked the changes from Egbert.